react/react1/week2/Todolist/src/components/todoAppList/todoAppList.jsx

```diff
@@ -0,0 +1,32 @@
+import TodoAppListItem from "../todoAppListItem/todoAppListItem";
+
+import './todoAppList.css';
+
+const TodoAppList = ({todos, onDelete, onToggleComplete}) => {
+    const elements = todos.map(item => {
+        const {id, ...itemProps} = item;
+        return (
+            <TodoAppListItem 
+                key={id}
+                {...itemProps}
+                onDelete={() => onDelete(id)}
+                onToggleComplete={() => onToggleComplete(id)}/>
+        )
+    })
+    let className = 'empty-list'
```

Comment:
This would have worked too:

`<h1 className={elements.length ? "empty-list" : "empty-list hide"}>"No items..."</h1>`

Or this:

```javascript
{!elements.length ? (
  h1 className="empty-list">"No items..."</h1>
) : (
  <ul className="list">
    {elements}
  </ul>
)}
```